plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/ShiftLeftReader.java

```diff
@@ -67,30 +68,30 @@ public TestSuiteResults parse(ResultFile resultFile) throws Exception {
         return tr;
     }
 
-    private int categoryToCWE(String category) {
+    private CweNumber categoryToCWE(String category) {
         switch (category) {
             case "cmdi":
-                return 78;
+                return CweNumber.OS_COMMAND_INJECTION;
             case "crypto":
-                return 327;
+                return CweNumber.WEAK_CRYPTO_ALGO;
             case "hash":
-                return 328;
+                return CweNumber.WEAK_HASH_ALGO;
             case "ldapi":
-                return 90;
+                return CweNumber.LDAP_INJECTION;
             case "pathtraver":
-                return 22;
+                return CweNumber.PATH_TRAVERSAL;
             case "securecookie":
-                return 614;
+                return CweNumber.INSECURE_COOKIE;
             case "sqli":
-                return 89;
+                return CweNumber.SQL_INJECTION;
             case "trustbound":
-                return 501;
+                return CweNumber.TRUST_BOUNDARY_VIOLATION;
             case "weakrand":
-                return 330;
+                return CweNumber.WEAK_RANDOM;
             case "xpathi":
-                return 643;
+                return CweNumber.XPATH_INJECTION;
             case "xss":
-                return 79;
+                return CweNumber.XSS;
             default:
                 throw new RuntimeException("Unknown category: " + category);
```

Comment:
Maybe change this to a simple system out println? Runtime exception seems harsh.

Reply:
I can change this, but I didn't touch this in my PR.